### PR TITLE
Import template via admin

### PR DESCRIPTION
**Pull request by hmpf**
- New button on the dmpt.Template changelist: "Import"
- New admin page "Import template", currently only from file, not url

### CODE REVIEW

**Comment by hmpf** [comment]
Missing: audit logging and logging to admin's log, exception handling if JSON of wrong format.

File: src/easydmp/dmpt/admin.py [hunk 1]
```diff
@@ -132,6 +144,79 @@ class TemplateAdmin(SetPermissionsMixin, ObjectPermissionModelAdmin):
...
+                try:
+                    serialized_dict = deserialize_template_export(template_export_jsonblob)
+                except TemplateImportError as e:
+                    error_msg = f'{e}, cannot import'
+                    messages.error(request, error_msg)
+                    return HttpResponseRedirect(url_on_error)
+
+                try:
+                    with warnings.catch_warnings(record=True) as w:
+                        tim = import_serialized_export(serialized_dict, via='admin')
```
Comment (vbhavdal) [inline src/easydmp/dmpt/admin.py:172]: It's a little bit nicer to either use an enum for these, or define consant for also 'admin', so it's less "hidden away".

**Review by vbhavdal — APPROVED** [review]

File: src/easydmp/dmpt/admin.py (hunk 1 above)
Comment (hmpf) [reply src/easydmp/dmpt/admin.py:172]: I have a branch for importing via API (and via fetching from an url), so I'll be doing something smarter then, I think.

**Comment by hmpf** [comment]
There ought to be more rigorous tests of both the import-mechanism and admin but as is the basic are covered.

**Pull request merged**